cmd/fyne/internal/templates/data/Makefile

```diff
@@ -5,15 +5,35 @@ LOCAL != test -d $(DESTDIR)/usr/local && echo -n "/local" || echo -n ""
 LOCAL ?= $(shell test -d $(DESTDIR)/usr/local && echo "/local" || echo "")
 PREFIX ?= /usr$(LOCAL)
 
+Name := {{ .Name | printf "%q" }}
+Exec := {{ .Exec | printf "%q" }}
+Icon := {{ .Icon | printf "%q" }}
+
 default:
+	# User install
+	# Run "make user-install" to install in ~/.local/
+	# Run "make user-uninstall" to uninstall from ~/.local/
+	#
+	# System install
 	# Run "sudo make install" to install the application.
 	# Run "sudo make uninstall" to uninstall the application.
```

Comment:
Should these perhaps be updated to say say something about it not being in the user folder? I can't think of any better wording myself. Any ideas?

Reply:
I changed this to use coma instead of slashes on https://github.com/fyne-io/fyne/pull/2915/commits/4e8eff78d743aed32ff4d970bcbe0395bea735ad - have I missed something ?

Reply:
Fixed again in https://github.com/fyne-io/fyne/pull/2915/commits/8824eefb90e687a204494c8420e2f2d11a6a2609

Reply:
For the wording, maybe:

```
# Run "sudo make install" to install the application for all users.
# Run "sudo make uninstall" to uninstall the application for all users.
```

That's a bit what proposes Windows for some softwares, you can install for the current user only or for all the users.

The problem here is that if a user installed "locally", and then it uses `sudo` to remove the package, it will believe that that will remove the package...

Reply:
Great. Thanks :)